tools/mo/openvino/tools/mo/back/preprocessing.py

```diff
@@ -328,7 +343,7 @@ def apply_preprocessing(ov_function: Model, argv: argparse.Namespace):
 
     mean_scale_values = update_mean_scale_to_dict(input_nodes=ov_function.inputs,
                                                   mean_scale_val=mean_scale_values,
-                                                  scale=argv.scale)
+                                                  scale=argv.scale if "scale" in argv else None)
```

Comment:
is this needed? Previously 'argv' always had 'scale' attribute and there was no need for such check. Is it not correct  anymore?

Reply:
you are right, will delete